app/dashboard/templates/dashboard/hackathon/hackathons.html

```diff
@@ -77,79 +181,178 @@
+                      <i class="fas fa-user-plus mr-2 d-none d-sm-inline"></i>
+                      Join
+                    </a>
+                  {% endif %}
+                  {% if event.hackathon.start_date|timesince >= "1 min" %}
+                    <a href="{% url 'hackathon_prizes' event.hackathon.slug %}" class="btn btn-gc-blue font-caption font-weight-semibold px-3 mb-2">
+                      <i class="fas fa-trophy mr-2 d-none d-sm-inline"></i>
+                      Prizes
+                    </a>
+                  {% endif %}
+                  {% if event.hackathon.show_results %}
+                    <a href="{% url 'hackathon_projects' event.hackathon.slug %}" class="btn btn-gc-blue font-caption font-weight-semibold px-3 mb-2">
+                      <i class="fas fa-dharmachakra mr-2 d-none d-sm-inline"></i>
+                      Projects
+                    </a>
+                  {% endif %}
+                </div>
+              </div>
+            </div>
+          </div>
+      {% endfor %}
+      </div>
+      </div>
+      {% endif %}
+      </div>
+
+      <div class="row my-4">
+        <div class="container">
+          <div class="row">
+            <h3 class="text-uppercase font-weight-bolder font-title ml-4">Hack with these tribes</h3>
+          </div>
+          <div class="row">
+            {% if tribes|length %}
+              {% for tribe in tribes %}
+                <div class="col-xs-12 col-sm-5 col-lg-3 text-center" align="center">
+                  <div class="mb-5 mt-5">
+                    <a href="{{ tribe.path }}" target="_blank" style="text-decoration:none;">
+                      <img width="128" src="{{ tribe.logo_path }}" alt="Tribe logo" />
+                      <span class="font-weight-bold d-flex align-items-center justify-content-around font-title text-center mt-1 mb-1" style="color:black;">{{ tribe.display_name }}</span>
+                    </a>
+                    <div calss="small"><i class="fas fa-user-alt mr-1"></i> {{ tribe.members }} members</div>
                   </div>
                 </div>
               {% endfor %}
+            {% endif %}
+          </div>
+          <div class="row text-center mb-5" align="center">
+            <div class="col-12"><a class="font-title" href="{% url 'tribes_directory' %}">See all tribes</a></div>
           </div>
         </div>
-      {% endif %}
-    {% endfor %}
+      </div>
+
+      <div class="row my-4">
+        <div class="container">
+            {% if workshops|length %}
+              <div class="row">
+                <h3 class="text-uppercase font-weight-bolder font-title ml-4">Learn through workshops</h3>
+              </div>
+              <div class="row mx-auto">
+                {% for workshop in workshops %}
+                    <div class="col-xs-12 col-sm-7 col-md-7 col-lg-5 col-xl-4">
+                      <div class="mb-5 mt-5">
+                        <div style="position:relative">
```

Comment:
bootstrap class can be used here `position-relative`

Reply:
@octavioamu refactored to use the bootstrap class.